week-1/interpret/time-format.js

```diff
@@ -14,16 +14,22 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// Ans. 7
 
 // b) How many function calls are there?
+// Ans. None
```

Comment:
Like in `percentage-change.js`, I think there's a function call in here 😉

Reply:
I can only think of one possibility again which is common here and in last one which is console.log again